src/web/components/Core/Banner.tsx

```diff
@@ -1,17 +1,28 @@
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import clsx from 'clsx';
 
 import './Banner.scss';
 
 type BannerProps = {
   message: string;
-  className?: string;
+  type: 'info' | 'warning' | 'error';
 };
 
-export function Banner({ message, className }: BannerProps) {
+export function Banner({ message, type }: BannerProps) {
```

Comment:
warning uses svg while the other two use font awesome. Not ideal but only the warning banner actually has a design for it, so we can always change this in future if we have a need for the other two.